preliz/tests/test_maxent.py

```diff
@@ -91,6 +92,8 @@
         (Normal(mu=0.5), -1, 1, 0.8, (-np.inf, np.inf), (0.581)),
         (Pareto(), 1, 4, 0.9, (1, np.inf), (1.660, 1)),
         (Pareto(m=2), 1, 4, 0.9, (2, np.inf), (3.321)),
+        (Rice(), 0, 4, 0.7, (0, np.inf), (0.020, 2.577)),
+        (Rice(nu=4), 0, 6, 0.9, (0, np.inf), (1.402)),
```

Comment:
(Rice(), 0, 4, 0.7, (0, np.inf), (0.027, 2.577)),
        (Rice(nu=4), 0, 6, 0.9, (0, np.inf), (1.402)),
```suggestion
        (Rice(), 0, 4, 0.7, (0, np.inf), (0.020, 2.577)),
        (Rice(nu=4), 0, 6, 0.9, (0, np.inf), (1.402)),
```